lib/Frontend/ParseableInterfaceModuleLoader.cpp

```diff
@@ -702,10 +742,17 @@ class ParseableInterfaceModuleLoaderImpl {
   // Check if all the provided file dependencies are up-to-date compared to
   // what's currently on disk.
   bool dependenciesAreUpToDate(ArrayRef<FileDependency> deps) {
+    SmallString<128> SDKRelativeBuffer;
     for (auto &in : deps) {
+      StringRef fullPath = in.getPath();
```

Comment:
Nitpick: This logic is repeated in `writeForwardingModule` -- does it make sense to move it into FileDependency/a helper?

Reply:
Moved it out into a helper